qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -67,6 +67,9 @@
             only Windows
             only default.default.with_cache.auto.default
             viofs_log_file_cmd = 'dir ${viofs_log_file}'
+        - with_packed_on:
+            only default.default.with_cache.auto.default
+            fs_driver_props = {"queue-size": 1024, "packed": "on"}
```

Comment:
Does this property support the qemu version `4.2.0` to the latest version?
Since I see the bug mentions it is only for the RHLE 9.

Reply:
Yes, it's only for rhel9. I added only Host_RHEL.m9 config, thanks.